dashboards/company/2015.erb

```diff
@@ -0,0 +1,23 @@
+<ul class="grid rows-2 cols-4" id="company">
+  <li class="row-1 col-1">
+    <div data-id="Lifetime-network-size" data-view="Number" data-unordered="true" data-title="<%= t 'company.network' %>" id="network-size" data-tooltip="<%= t 'company.tips.network' %>"></div>
```

Comment:
So… Using Lifetime-network-size here matches the text in company.tips.network, but is almost certainly wrong.

I think what we probably want here is the cumulative network size up to 2015. If so, that will be easier to do once theodi/odi-metrics-tasks#37 is merged.